ciao-launcher/main.go

```diff
@@ -341,6 +343,54 @@ func printClusterConfig() {
 	glog.Infof("Memory Limit:         %v", memLimit)
 }
 
+func discoverCertificate(certPath, certPrefix string) (string, error) {
+	ciaoCertsPath := "/etc/pki/ciao/"
+	fmt.Println("discoverCertificate")
+	if _, err := os.Stat(certPath); os.IsNotExist(err) {
+		files, err := ioutil.ReadDir(ciaoCertsPath)
+		if err != nil {
+			return certPath, err
```

Comment:
@sameo @mcastelino it wouldn't return "", it will return the default one that is specified at: https://github.com/01org/ciao/blob/master/ciao-launcher/main.go#L98-99.

This error's return will happen when the ``/etc/pki/ciao`` dir doesn't exist.